### PR TITLE
[IN PROGRESS] UI2 changes

### PR DESCRIPTION
**Pull request by IvRRimum**
- [x] Replace favicons(without white background)
- [x] Video item design
- [x] No videos in my feed yet 
- [x] Change font to Montserrat
- [x] No subscriptions yet
- [x] Click on logo, take to root path
- [x] Move donations and github link to fixed location at the bottom right of the screen.
- [x] Make logout button work
- [x] If a person is not logged in dont show Feeds and subscriptions nav items
- [x] Add notifications( on subscription add) use this https://github.com/kyvg/vue3-notification ?
- [x] login button when not logged in, logout when logged in
- [x] Feeds - filters by the design
- [x] Subscriptions - Import/export by the design
- [x] Add search to pages(trending, feed, channel view, search view)
- [x] Fix autocomplete location
- [x] Search page filters by design
- [x] Fix infinite scroll in search, channel view, feeds pages
- [x] notification popup not seeable on mobile devices
- [x] Save the status of menu bar(collapse/expand) in browser memory
- [x] Video comments infinity scrool

### CODE REVIEW

**Comment by IvRRimum** [comment]
@FireMasterK Are we good to merge?

**Comment by IvRRimum** [comment]
Wait, no. Need to fix infinite scrool for comments

**Comment by IvRRimum** [comment]
@FireMasterK Done

File: src/components/WatchVideo.vue [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <div v-if="video && !isEmbed" class="uk-container uk-container-xlarge">
         <ErrorHandler v-if="video && video.error" :message="video.message" :error="video.error" />
 
-        <div v-show="!video.error">
+        <div v-show="!video.error" id="video-and-info">
```
Comment (FireMasterK) [inline src/components/WatchVideo.vue:17]: Instead of using id, and document.getElement... You should use `ref="name"` and `this.$refs.name`

**Review by FireMasterK — CHANGES_REQUESTED** [review]

**Comment by VarLad** [comment]
This needs to implement Chapters now as well I guess? 
@FireMasterK @IvRRimum

**Comment by Bnyro** [comment]
See #1391 for Piped's future UI

**Pull request closed without merging**